docs/samples/outlier-detection/alibi-detect/cifar10/.gitignore

```diff
@@ -0,0 +1,5 @@
+event-display.yaml
+broker.yaml
+trigger.yaml
+cifar10od.yaml
+cifar10.yaml
```

Comment:
why need `gitignore` here ? I see some of the files are used.

Reply:
All the files should be written when running the notebook so none should be stored in git.

Reply:
for the outlier-detection example the yamls are still there